src/jv.ml

```diff
@@ -222,13 +222,9 @@ external to_error : t -> Error.t = "%identity"
 
 let throw ?name msg =
   let e = Error.v ?name msg in
-  (js_expr "(function (exn) { throw exn })" : jv -> 'a) e
+  (js_expr "(function (exn) { throw exn })" : Error.t -> 'a) e
 
 exception Error of Error.t
```

Comment:
I think you forgot to equate this with `Jsoo_runtime.Error.Exn`

(dont' bother I will amend the patch myself)